aio/src/app/shared/reporting-error-handler.ts

```diff
@@ -19,12 +19,29 @@ export class ReportingErrorHandler extends ErrorHandler {
    * @param error Information about the error.
    */
   handleError(error: any) {
+    const versionedError = this.prefixErrorWithVersion(error);
+
     try {
-      super.handleError(error);
+      super.handleError(versionedError);
     } catch (e) {
       this.reportError(e);
     }
-    this.reportError(error);
+    this.reportError(versionedError);
+  }
+
+  private prefixErrorWithVersion<T>(error: T): T {
+    const prefix = `[v${VERSION.full}] `;
+
+    if (error instanceof Error) {
+      const oldMessage = error.message;
+      error.message = `${prefix}${oldMessage}`;
```

Comment:
NIT: using backtick strings seems a bit unnecessary here:

```suggestion
      error.message = prefix + oldMessage;
````

Or am I missing something?

Reply:
Ha! I knew you would say that 😛
No, it's not necessary to use backticks. Happy to switch to string concatenation.